devmand/gateway/src/devmand/devices/cli/StructuredUbntDevice.h

```diff
@@ -25,7 +25,7 @@ class StructuredUbntDevice : public Device {
  private:
   Command createInterfaceCommand(string name, bool enabled);
```

Comment:
This can be omitted form .h and implemented just as a static function (like all the parsing functions)

Reply:
sure, idc, seems the same to me